docs/docs/v4/image-coil/README.md

```diff
@@ -0,0 +1,35 @@
+# Image Coil
+
+<MavenBadge4 :artifact="'image-coil'" />
+
+Image loading based on `Coil` library
+
+```kt
```

Comment:
Unfortunately syntax highlighting won't work for `kt`, please change to `kotlin`